server/plugin.go

```diff
@@ -194,6 +194,11 @@ func (p *Plugin) createPost(userID, channelID, message string) {
 			"attachments": []*model.SlackAttachment{slackAttachment},
 		},
 	}
+
+	if fileIds != nil {
+		post.FileIds = append(post.FileIds, fileIds[0])
```

Comment:
Does it make sense to consider all the fieldIDs that are delivered, and not only the first one?

Reply:
For now, it seems like it's not necessary to consider all fileIDs as only one is being delivered. Keeping it this way might clarify the purpose of including the variadic parameter.

Reply:
I was thinking the same, but since there is no code passing more than one at the moment, I think this makes sense to just post one.

But now that I read this code again, I think `len(fileIds) > 0` is a more readable way to check the presence of the file ID. Go implicitly checks if it is nil in the process. @DanielSz50 what do you think about making this change?

Reply:
I think you're right, done.